4MD/src/main/java/com/infoshareacademy/four_md/models/User.java

```diff
@@ -0,0 +1,51 @@
+package com.infoshareacademy.four_md.models;
+import java.util.List;
+import java.util.UUID;
+
+public class User {
+    private String id;
+    private String name;
+    private String surname;
+    private List<Recipe> listOfRecipes;
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId() {
+        UUID uuid = UUID.randomUUID();
+        this.id = uuid.toString();
+    }
```

Comment:
lepiej by było użyć inta jako id dla spójności

Reply:
UUID składa się z 16 bajtów, czyli z 128 bitów i nie da się tego zamienić na inta, ponieważ int ma tylko 32 bity.

Reply:
to użyj inta zamiast UUID (jakbgybyś nie zrozumiał)

Reply:
Mój saver tego nie przyjmie

Reply:
Okej wszystko już wyjaśniłem z Grzegorzem, jutro to poprawię jako zwykły int.